src/sage/schemes/elliptic_curves/hom_composite.py

```diff
@@ -893,3 +896,291 @@ def inseparable_degree(self):
             1331
         """
         return prod(phi.inseparable_degree() for phi in self._phis)
+
+    def is_cyclic(self, _check_j=True, _reduce_kernel=True):
+        r"""
+        Determine whether the isogeny is cyclic (separable with cyclic kernel).
+
+        ALGORITHM:
+
+        For each prime $\ell$ with $\ell^2$ dividing the isogeny degree,
+        test that $\ell$-torsion is not fully nullified. This is done by
```

Comment:
```suggestion
        For each prime `\ell` with `\ell^2` dividing the isogeny degree,
        test that `\ell`-torsion is not fully nullified. This is done by
```
There are a few more in this docstring, I won't comment on them.